StyleCop/syncSettingsOverEveryDevice.ps1

```diff
@@ -1,7 +1,7 @@
 $ErrorActionPreference = 'Stop'
 
 #Whitelisted projects
-$projectWhiteList = ("AD5328", "4Relay", "Ads1115", "Adxl345", "Adxl357", "Ags01db", "Ahtxx")
+$projectWhiteList = ("ePaper")
```

Comment:
This is replacing a "global" PS1 script... probably not intended ....

Reply:
Up to you, I'm good with it.